src/pages/events/2022/07/london-assurex-sitecore-usergroup.md

```diff
@@ -14,16 +14,34 @@ venue:
   details:
 agenda:
   - agenda-item:
-    time: "18:30"
+    time: "18:00"
     value: Arrival, drinks and networking
+  - talk:
+    time: "18:45"
+    who: "Saurabh Sachdeva - Founder @ Assurex"
+    intro: "Welcome and kick-off"
+    description:
+  - talk: 
+    time: "19:00"
+    who: "Timothy March - Solution Architect II @ EPAM Systems"
+    intro: "Getting started with Sitecore Content Publisher"
+    description: "In this session, Timothy will talk about Introduction to Sitecore Content Publisher and a live demo of the tool, API Integration options, and extension possibilities."
```

Comment:
How about "will give an introduction..."

Reply:
Changed the words